### ISSUE TITLE
SEToolbox can't locate Application anymore. Program is unusable.

### ISSUE THREAD

**Issue by virion88** [midspace/SEToolbox#187]
- [ ] Ensure you have installed the latest version of SEToolbox installed. Visit [latest release](https://github.com/midspace/SEToolbox/releases/latest) to verify.
- [ ] Ensure that the issue is reproducible for testing. You may be requested to provide a link to a test world if we cannot reproduce it with your information.
- [ ] Retrieve the Event Log from when SEToolbox crashed. Read the following [instructions](https://github.com/midspace/SEToolbox/wiki/Reporting-bugs) on retrieving the event log.

**SEToolbox Version:** 01.191.20 Release 2
**SE Version:** 1.193.022

![Anmerkung 2019-12-16 174460](https://user-images.githubusercontent.com/29334879/70925671-eff95980-202b-11ea-8177-e5a20bd5618c.png)

![Anmerkung 2019-12-16 174459](https://user-images.githubusercontent.com/29334879/70925680-f2f44a00-202b-11ea-8104-0343c8e07074.png)

When selecting "Space Engineers.exe", SEToolbox won't accept it as a location.
I already tried everything. Updating SEToolbox, reinstalling Space Engineers, nothing.

I can't use SEToolbox at all anymore since it won't run without the EXE located.
Is there a solution to that?

**Comment by minestargate** [comment]
no way.you must use SE Version: 1.190!

**Comment by Scrappy1989** [comment]
use it with SE 1.191

**Comment by MrEdinLaw** [comment]
No version works for me...

**Comment by NathanHawks** [comment]
Or I guess give up on space engineers because they slapped a 1.0 on a lemon :(

**Comment by NathanHawks** [comment]
Are you saying downgrade the game? How is that going to play out for our save files?

**Comment by NathanHawks** [comment]
So does that mean there are no plans to continue updating SEToolbox?

**Comment by kontu** [comment]
> So does that mean there are no plans to continue updating SEToolbox?

Correct, as per the main page for this project: "Currently dead as of 2019-08-23 due to changes in the major game update containing Economy."

**Comment by uselessfuck** [comment]
Goddamn it, that's a real shame

**Comment by kontu** [comment]
> Goddamn it, that's a real shame

Good news - https://github.com/mmusu3/SEToolbox as a fork works on the newest version!

**Comment by xMiracle45** [comment]
> > Goddamn it, that's a real shame
> 
> Good news - https://github.com/mmusu3/SEToolbox as a fork works on the newest version!

This does not work either
<img width="206" alt="Annotation 2020-08-10 195843" src="https://user-images.githubusercontent.com/69490457/89845502-4760b280-db44-11ea-812a-12059fb590d6.png">

**Comment by DereC4** [comment]
It's fixed I believe

**Comment by Omiyaru** [comment]
this issue is happening again

**Comment by SpadesNeil** [comment]
Someone already linked it but I'm going to put in big bold letters for people like me who missed it the first time.

# Hey you, if you're reading this, and you just arrived here off Google and are now wondering why the save editor is busted, this version of SEToolbox is abandoned. Go download this other version.

https://github.com/mmusu3/SEToolbox/releases/tag/v1.205.026.1
